.gitignore

```diff
@@ -5,3 +5,4 @@ develop-eggs/
 dist/
 .DS_Store
 *.egg-info/
+.ipynb_checkpoints/
```

Comment:
I guess it is something to easy the development work when trying this from the repo...